src/main/java/ui/TestController.java

```diff
@@ -65,6 +66,14 @@ public static boolean isTestGlobalConfig() {
         return hasUI() && commandLineArgs.getOrDefault("testconfig", "false").equalsIgnoreCase("true");
     }
 
+    /** Used for testing startup board on launch (i.e. when the number of stored repos is 0)
+     * StartBoardLauncherTest makes use of this argument to test the launch of the board for first time users
+     */
+    public static boolean isTestStartupBoard() {
```

Comment:
Again, we shouldn't refer to how methods are used. The description should start on the second line, as a matter of style.

Also, the method name isn't very clear, even though I understand what it does from the method body. `shouldTestStartupBoard`?